swyft/store/simulator.py

```diff
@@ -102,6 +102,7 @@ def from_command(
         sim_shapes: SimShapeType,
         set_input_method: Callable,
         get_output_method: Callable,
+        shell: bool = False,
```

Comment:
I see your concern. However, I think that this might be quite useful in practice, as users who run command-line programs on HPC systems typically need to load specific environments/modules to run their applications. So running commands such as `module load mylibrary && cd ${TMPDIR} && myprogram.x ` becomes straightforward if one can set `shell` to `True`.

To make users aware of the risk, we have also linked the security considerations you cite in the docstring:
https://github.com/undark-lab/swyft/blob/1daab2f945f191ebabb61c5300cc950ff0a2b482/swyft/store/simulator.py#L126